src/main/java/no/digipost/api/client/representations/Document.java

```diff
@@ -82,6 +81,13 @@
     @XmlElement(name = "content-hash", nillable = false)
     protected ContentHash contentHash;
 
+    @XmlElementWrapper(name="metadata")
+    @XmlElements({
+            @XmlElement(name = "residence", type = Residence.class, namespace = "http://api.digipost.no/schema/datatypes"),
+            @XmlElement(name = "appointment", type = Appointment.class, namespace = "http://api.digipost.no/schema/datatypes")
+    })
+    protected List<DataType> metadata;
```

Comment:
Kult med sånn elementwrapper-annotering! Hadde det vært mulig å bare referere til namespace i stedet for konkrete klasser? Så slipper vi å huske oppdatere denne klassen hver gang en ny metadatatype kommer?

Regner med det da må gjøre globalt i JAXB-context